troposphere/static/js/components/projects/resources/instance/details/sections/InstancePlaybookHistorySection.jsx

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import subscribe from "utilities/subscribe";
+import PlaybookStatus from "./PlaybookStatus";
+
+const InstancePlaybookHistorySection = React.createClass({
+    displayName: "InstancePlaybookHistorySection",
+
+    style() {
+        return {
+            td: {
+                wordWrap: "break-word",
+                whiteSpace: "normal"
+            }
+        }
+    },
+    getInitialState() {
+        return {
+            refreshing: false
+        }
+    },
+
+    renderHeaderRow: function() {
+        return (
+            <tr>
+                <th style={{ width: "200px"}}>Command</th>
+                <th>Arguments</th>
+                <th style={{ width: "70px"}}>Status</th>
+                <th>Message</th>
+            </tr>
+        );
+    },
+    renderInstancePlaybookHistoryRow: function(instancePlaybookHistory) {
+        let { td } = this.style();
+        let playbook_args = instancePlaybookHistory.get("playbook_arguments");
+        return (
+        <tr key={instancePlaybookHistory.id}>
+            <td style={td}>
```

Comment:
Using the "card" classes for now is ok. I'd like to make this into a component in CY-UI in the near future though.